### PR TITLE
Add filterfalse to itertools

### PR DESCRIPTION
**Pull request by ziima**

### CODE REVIEW

**Comment by maxfischer2781** [comment]
Thanks for the contribution and especially for spotting that this was missing!

**Review by maxfischer2781 — CHANGES_REQUESTED** [review]
This looks overall good to go. Please take a look at the failed "Static Checks" and adjust the code accordingly.

Feel free to disregard the codecov failure. I'll leave it up to you whether you want to address the CodeQL report, I'm fine with both variants.

* [x] Static Checks must succeed
* [x] Consider to address the CodeQL report

File: unittests/test_itertools.py [hunk 1]
```diff
@@ -178,6 +178,34 @@
     )
 
 
+filterfalse_cases = (
+    (lambda x: True, [0, 1] * 5),
+    (lambda x: False, [0, 1] * 5),
+    (lambda x: bool(x), [0, 1] * 5),
```
[the remark this answers is not in the document]
Comment (ziima) [reply unittests/test_itertools.py]: Fixed.

**Comment by ziima** [comment]
>     * [ ]  Static Checks must succeed
>     * [ ]  Consider to address the CodeQL report

Both should be fixed now.

**Review by maxfischer2781 — APPROVED** [review]
Looks good to go!

**Pull request merged**